UPGRADE-1.13.md

```diff
@@ -450,70 +682,153 @@ List of affected classes:
+1. New parameters have been added to specify the validation groups for a given shipping method rule and calculator.
+   If you have any custom validation groups for your calculators or rules, you need to add them to
+   your `config/packages/_sylius.yaml` file.
+   Also, if you have your own shipping method rule or calculator and want to add your validation groups you can add
+   another key to the `validation_groups` parameter:
 
-1. The first parameter of the constructor in the `\Sylius\Component\Core\Promotion\Checker\Rule\ItemTotalRuleChecker` class has been deprecated and will be removed in version 2.0.
+    ```yaml
+    sylius_shipping:
+        shipping_method_rule:
+            validation_groups:
+                total_weight_greater_than_or_equal:
+                    - 'sylius'
+                    - 'sylius_shipping_method_rule_total_weight'
+                order_total_greater_than_or_equal:
+                    - 'sylius'
+                    - 'sylius_shipping_method_rule_order_total'
+                your_shipping_method_rule:
+                    - 'sylius'
+                    - 'your_custom_validation_group'
+        shipping_method_calculator:
+            validation_groups:
+                flat_rate:
+                    - 'sylius'
+                    - 'sylius_shipping_method_calculator_rate'
+                per_unit_rate:
+                    - 'sylius'
+                    - 'sylius_shipping_method_calculator_rate'
+                your_shipping_method_calculator:
+                    - 'sylius'
+                    - 'your_custom_validation_group'
+    ```
 
-1. The service definition for `sylius.promotion_rule_checker.item_total` has been updated. The class has been changed from `Sylius\Component\Promotion\Checker\Rule\ItemTotalRuleChecker` to `Sylius\Component\Core\Promotion\Checker\Rule\ItemTotalRuleChecker`.
+# State Machine
 
-1. Sylius Mailer email configuration keys in the `src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius/sylius_mailer.yml` file have been changed:
+1. We have configured all existing Sylius graphs to be usable with the Symfony Workflow out of the box.
+   Winzou State Machine is still the default state machine for all graphs, but you can switch to Symfony Workflow via
+   configuration:
 
-    Deprecated:
     ```yaml
-    sylius_mailer:
-        emails:
-           account_verification_token:
-               subject: sylius.emails.user.verification_token.subject
+    sylius_state_machine_abstraction:
+        graphs_to_adapters_mapping:
+            <graph_name>: <adapter_name>
+            # e.g.
+            sylius_order_checkout: symfony_workflow # available adapters: symfony_workflow, winzou_state_machine
+   
+        # we can also can the default adapter
+        default_adapter: symfony_workflow # winzou_state_machine is a default value here
     ```
+   Starting with `Sylius 2.0` Symfony Workflow will be the default state machine for all graphs.
```

Comment:
Actually, that paragraph is false. In `1.13` we switched all internal state machines to Workflow, with a possibility to revert that. Starting with `Sylius 2.0` we'll offer only Symfony Workflow and (probably) we'll remove the abstraction.